# References

Complete reference list for the VecMap manuscript:

1. [Bray *et al.*, 2016](https://doi.org/10.1038/nbt.3519) - Near-optimal probabilistic RNA-seq quantification. *Nature Biotechnology*, 34(5), 525-527.

2. [Clement *et al.*, 2019](https://doi.org/10.1038/s41587-019-0032-3) - CRISPResso2 provides accurate and rapid genome editing sequence analysis. *Nature Biotechnology*, 37(3), 224-226.

3. [Dobin *et al.*, 2013](https://doi.org/10.1093/bioinformatics/bts635) - STAR: ultrafast universal RNA-seq aligner. *Bioinformatics*, 29(1), 15-21.

4. [Harris *et al.*, 2020](https://doi.org/10.1038/s41586-020-2649-2) - Array programming with NumPy. *Nature*, 585(7825), 357-362.

5. [Langmead *et al.*, 2009](https://doi.org/10.1186/gb-2009-10-3-r25) - Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. *Genome Biology*, 10(3), R25.

6. [Li & Durbin, 2009](https://doi.org/10.1093/bioinformatics/btp324) - Fast and accurate short read alignment with Burrows-Wheeler transform. *Bioinformatics*, 25(14), 1754-1760.

7. [Li, 2018](https://doi.org/10.1093/bioinformatics/bty191) - Minimap2: pairwise alignment for nucleotide sequences. *Bioinformatics*, 34(18), 3094-3100.

8. [Li *et al.*, 2014](https://doi.org/10.1186/s13059-014-0554-4) - MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. *Genome Biology*, 15(12), 554.

9. [Luo *et al.*, 2013](https://doi.org/10.1371/journal.pone.0065632) - FM-index based algorithm for mapping short reads. *PLOS ONE*, 8(5), e65632.

10. [Patro *et al.*, 2017](https://doi.org/10.1038/nmeth.4197) - Salmon provides fast and bias-aware quantification of transcript expression. *Nature Methods*, 14(4), 417-419. 